backend/nodes/ncnn_nodes.py

```diff
@@ -288,13 +287,14 @@ def __init__(self):
 
     def perform_interp(self, bin_a: np.ndarray, bin_b: np.ndarray, amount: int):
         try:
-            amount_a = amount / 100
-            amount_b = 1 - amount_a
+            amount_a = amount / 100.0

```

Comment:
Does this even matter?

```py
>>> 4 / 100
0.04
>>> 4 / 100.0
0.04
```

Reply:
Nah, just didn't revert it after actually fixing it